packages/cypher-builder/src/clauses/Match.ts

```diff
@@ -65,11 +75,25 @@ export class Match<T extends MatchableElement> extends Clause {
         return this;
     }
 
+    /** Makes the clause an OPTIONAL MATCH
+     * @see [Cypher Documentation](https://neo4j.com/docs/cypher-manual/current/clauses/optional-match/)
+     * @example
+     * ```ts
+     * new Match(new Node({labels: ["Movie"]})).optional();
```

Comment:
Would this not always be used like:

```suggestion
     * new CypherBuilder.Match(new Node({labels: ["Movie"]})).optional();
```

Reply:
Close :smile: 
```suggestion
     * new Cypher.Match(new Node({labels: ["Movie"]})).optional();
```